javascript/javascript1/week3/savenote.js

```diff
@@ -0,0 +1,37 @@
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id
+  });
+  return notes;
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+//////////////////getNote////////////
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id) {
+      return notes[i];
+    }
+
+    }
+    if (notes[i].id !== 'number') {
```

Comment:
this check should be put before the get is executed. 

in case i input 'blablastring' and the id is actually there it will return before checking the value is not a number.

Reply:
Got it. Changed now